jd-geek-source/src/app/interceptors/jwt.interceptor.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import {
+  HttpRequest,
+  HttpHandler,
+  HttpEvent,
+  HttpInterceptor,
+} from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class JwtInterceptor implements HttpInterceptor {
+  intercept(
+    request: HttpRequest<any>,
+    next: HttpHandler
+  ): Observable<HttpEvent<any>> {
+    const skipJWT = request.headers.get('skipJWT');
```

Comment:
doing anything with this?

Reply:
I need to review this more.  I don't think I am  actually using this.  I put it in trying troubleshoot getting data back from the BB API.